src/main/scala/com/codacy/api/dtos/Language.scala

```diff
@@ -0,0 +1,260 @@
+                               Haskell,
+                               Erlang,
+                               YAML,
+                               Dart,
+                               Elm,
+                               HTML,
+                               Groovy,
+                               VisualForce,
+                               Perl,
+                               CSharp,
+                               VisualBasic,
+                               ObjectiveC,
+                               FSharp,
+                               Cobol,
+                               Fortran,
+                               R,
+                               Scratch,
+                               Lua,
+                               Lisp,
+                               Prolog,
+                               Julia,
+                               Kotlin,
+                               OCaml)
+
+  // Support startdate: always
+  case object Javascript extends Language(extensions = Set(".js", ".jsx"))
+
+  case object Scala extends Language(extensions = Set(".scala"))
+
+  case object CSS extends Language(extensions = Set(".css"))
+
+  case object PHP extends Language(extensions = Set(".php"))
+
+  case object Python extends Language(extensions = Set(".py"))
+
+  // Support startdate: 31 August 2015
+  case object Ruby
+      extends Language(extensions = Set(".rb", ".gemspec", ".podspec", ".jbuilder", ".rake", ".opal"),
+                       files = Set("Gemfile",
+                                   "Gemfile.lock",
+                                   "config.ru",
+                                   "Rakefile",
+                                   "Capfile",
+                                   "Guardfile",
+                                   "Podfile",
+                                   "Thorfile",
+                                   "Vagrantfile",
+                                   "Berksfile",
+                                   "Cheffile",
+                                   "Vagabondfile",
+                                   "Fastfile"))
+
+  case object Java extends Language(extensions = Set(".java"))
+
+  case object CoffeeScript extends Language(extensions = Set(".coffee"))
+
+  // Support startdate: 27 June 2016
+  case object Swift extends Language(extensions = Set(".swift"))
+
+  // Support startdate: 18 July 2016
+  case object CPP extends Language(extensions = Set(".cpp", ".hpp", ".cc", ".cxx"))
+
+  case object C extends Language(extensions = Set(".c", ".h"))
+
+  case object Shell extends Language(extensions = Set(".sh", ".bash"))
+
+  // Support startdate: 24 November 2016
+  case object TypeScript extends Language(extensions = Set(".ts", ".tsx"))
+
+  // Support startdate: December 2016
+  case object Dockerfile extends Language(extensions = Set(".dockerfile"), files = Set("Dockerfile"))
+
+  // Support startdate: January 2017
+  case object SQL extends Language(extensions = Set(".sql"))
```

Comment:
Since we now support two languages in tools I brought SQL back to its own language. Do you think this is a problem?